modules/command/dh/dh-offer-finalized-command.js

```diff
@@ -30,7 +30,8 @@ class DhOfferFinalizedCommand extends Command {
                 const {
                     offerId: eventOfferId,
                 } = JSON.parse(e.data);
-                return offerId === eventOfferId;
+                this.logger.important(`${offerId}, ${eventOfferId}`);
```

Comment:
Can we add a full sentence, like: "Offer with ${offerId} and ${eventOfferId} got finalised" or similar because full sentence will improve clarity of existing log line?

Reply:
It's down bellow. This line mistakenly got here. It was needed for the debug.

Reply:
Actually it's in different command and it looks like:
```
important: Offer 0x698e07e2257b5e6019d2b06b87fc0007bf1fc83ddbdc1a19502746f6e4e8e09d finalized
```